move/sources/canvas/token.move

```diff
@@ -195,40 +172,30 @@ module addr::canvas_token {
     public entry fun create(
         caller: &signer,
         // Arguments for the token + object.
-        description: String,
         name: String,
+        description: String,
         // Arguments for the canvas. For now we don't allow setting the palette
         // because it is a pain to express vector<Color> in an entry function.
-        width: u64,
-        height: u64,
-        per_account_timeout_s: u64,
-        can_draw_for_s: u64,
-        cost: u64,
-        cost_multiplier: u64,
-        cost_multiplier_decay_s: u64,
+        width: u16,
+        height: u16,
+        per_account_timeout_s: u8,
```

Comment:
This is only 255 at most, maybe this should be u16 so we can go higher than that.

Reply:
updated